### DISCUSSION TITLE
rectangle  mouse  line cross

### DISCUSSION THREAD

**Discussion by monkeycc** [wkentaro/labelme#1051] [Ideas / Feature requests]
rectangle

I want the mouse to show a cross

Convenient for me to locate

<img width="408" alt="微信截图_20220602200358" src="https://user-images.githubusercontent.com/6490927/171625270-8646ab55-a5d3-44df-a935-279a72cb156a.png">

**Comment by monkeycc** [comment]
How to achieve
Has anyone studied it

**Comment by wkentaro** [comment]
Check this PR: https://github.com/wkentaro/labelme/pull/1214
![image](https://user-images.githubusercontent.com/4310419/201517235-393bb3c2-cfae-48b4-a8db-4c596544b894.png)

**Comment by monkeycc** [comment]
Thanks for this feature. It's very practical
